### PR TITLE
Test the force computation

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: physics/geopotential_test.cpp [hunk 1]
```diff
@@ -549,6 +552,249 @@ TEST_F(GeopotentialTest, ThresholdComputation) {
...
+  auto const get_longitudinal_acceleration =
+      [&direction, &get_acceleration](Geopotential<ICRS> const& geopotential,
+                                      Length const& r) {
+        Vector<double, ICRS> const down = -direction;
+        Bivector<double, ICRS> const north({0, 0, 1});
+        Vector<double, ICRS> local_east = down * north;
+        return InnerProduct(get_acceleration(geopotential, r), local_east);
+      };
+
+  // Above the  outer threshold for J2.
```
Comment (pleroy) [inline physics/geopotential_test.cpp]: Extra space.

File: physics/geopotential_test.cpp [hunk 2]
```diff
@@ -549,6 +552,249 @@ TEST_F(GeopotentialTest, ThresholdComputation) {
...
+  EXPECT_THAT(
+      get_acceleration(earth_geopotential, 5'000'000 * Kilo(Metre)).Norm(),
+      Eq(0 / Pow<2>(Metre)));
+
+  {
+    // Inspect the J2 sigmoid.
+    Length const s0 = earth_geopotential.degree_damping()[2].inner_threshold();
+    Length const s1 = earth_geopotential.degree_damping()[2].outer_threshold();
+    EXPECT_THAT(s0, IsNear(1'500'000 * Kilo(Metre)));
```
Comment (pleroy) [inline physics/geopotential_test.cpp]: Check `s1` too, even if it's a fixed factor of 3 at the moment.  Same comment at other places below.

File: physics/geopotential_test.cpp [hunk 3]
```diff
@@ -549,6 +552,249 @@ TEST_F(GeopotentialTest, ThresholdComputation) {
...
+  // Below the inner threshold for J2, but still above all other outer
+  // thresholds.
+  EXPECT_THAT(earth_geopotential.degree_damping()[2].inner_threshold(),
+              Gt(1'000'000 * Kilo(Metre)));
+  EXPECT_THAT(earth_geopotential.tesseral_damping().outer_threshold(),
+              Lt(1'000'000 * Kilo(Metre)));
+  EXPECT_THAT(earth_geopotential.degree_damping()[3].outer_threshold(),
+              Lt(1'000'000 * Kilo(Metre)));
+  EXPECT_THAT(get_acceleration(earth_geopotential, 1'000'000 * Kilo(Metre)),
+              Eq(get_acceleration(*geopotential_j2, 1'000'000 * Kilo(Metre))));
```
Comment (pleroy) [inline physics/geopotential_test.cpp]: Use `AlmostEquals` with a tolerance.

File: physics/geopotential_test.cpp [hunk 4]
```diff
@@ -549,6 +552,249 @@ TEST_F(GeopotentialTest, ThresholdComputation) {
...
+         get_longitudinal_acceleration(*geopotential_degree[2],
+                                       (s0 + s1) / 2)) /
+            get_longitudinal_acceleration(*geopotential_degree[3],
+                                          (s0 + s1) / 2),
+        AlmostEquals(0.5, 2130));
+  }
+
+  // The outer threshold for degree 5 is above the inner threshold for degree 3,
+  // so degrees 4 and above have mixed sigmoids, which are tricky to test.
+    EXPECT_THAT(
```
Comment (pleroy) [inline physics/geopotential_test.cpp]: Bad indent.

File: physics/geopotential_test.cpp (hunk 3 above)
Comment (eggrobin) [reply physics/geopotential_test.cpp]: No, this is equality because the same path is taken, not because different calculations give the same result.

**Pull request merged**